css/emoji-mart.css

```diff
@@ -334,7 +373,7 @@
 
 .emoji-mart-skin-swatches.custom.opened .emoji-mart-skin-swatch.custom {
   width: 36px;
-  height: 38px;
+  height: 36px;
```

Comment:
adding this to avoid this override:

https://github.com/goabstract/ui/blob/be219c88cdc2618ac6db62c71e436f4244ae4844/core/src/components/EmojiPicker/style.scss#L111-L116